### ISSUE TITLE
Do we really need extension configuration export?

### ISSUE THREAD

**Issue by AllenJB** [wdhongtw/markdown-link#6]
It appears the latest version of this extension wants permission to manage my downloads. Why does it want this?

This permission seems completely unnecessary for its advertised purpose.

**Comment by wdhongtw** [comment]
It's related the new link-transform feature introduced by #1.

The extension use download API to implement config-export action, (see [options.js](https://github.com/wdhongtw/markdown-link/blob/master/options.js#L151))

**Comment by wdhongtw** [comment]
I will leave this issue open, and use this comment for a short-term survey

- If you like this change, please click 👍
- If you don't feel good about this change, plase click 👎 

If lots of people don't like this, maybe I should release v0.3.1 and remove the configuration import/export feature.  :(

**Comment by AllenJB** [comment]
Thanks for the response (and the extension). That's fine. I'm just wary of browser extensions requesting permissions that they don't obviously need.

I think it would be useful to users if you updated the store description to include why this permission is requested.

**Comment by wdhongtw** [comment]
> I think it would be useful to users if you updated the store description to include why this permission is requested.

Yes, that was a mistake, I remember that I do update the description, but nothing changed, maybe I opened multiple tabs and edit in the wrong place. :(

Anyway, I decide to remove the import / export functionality now. I did not realize that it's actually trigger a [warning](https://developer.chrome.com/docs/extensions/mv3/permission_warnings/#permissions_with_warnings) for all users. I think we can add this feature back once there are mature requirement from users.
